packages/daimo-api/src/contract/noteIndexer.ts

```diff
@@ -51,45 +60,58 @@ export class NoteIndexer {
         where block_num >= $1
         and block_num <= $2
         and chain_id = $3
+        order by (block_num, tx_idx, log_idx) asc
```

Comment:
👍

Reply:
note index interacts poorly w reorgs, but i don't think that's a blocker currently.

Reply:
seq deleted